### ISSUE TITLE
Qpdf error

### ISSUE THREAD

**Issue by dustinromey** [skycocker/chromebrew#1764]
## Description
Receive an error when trying to run qpdf

If you executed a specific command:
```shell
qpdf --empty --pages *.pdf -- out.pdf
```

Output:
```shell
qpdf: error while loading shared libraries: libqpdf.so.18: cannot open shared object file: No such file or directory
```

## Versions and system information
chromebrew: `0.4.3`

ChromeOS: `
CHROMEOS_ARC_ANDROID_SDK_VERSION=25
CHROMEOS_ARC_VERSION=4524559
CHROMEOS_AUSERVER=https://tools.google.com/service/update2
CHROMEOS_BOARD_APPID={01906EA2-3EB2-41F1-8F62-F0B7120EFD2E}
CHROMEOS_CANARY_APPID={90F229CE-83E2-4FAF-8479-E368A34938B1}
CHROMEOS_DEVSERVER=
CHROMEOS_RELEASE_APPID={01906EA2-3EB2-41F1-8F62-F0B7120EFD2E}
CHROMEOS_RELEASE_BOARD=eve-signed-mpkeys
CHROMEOS_RELEASE_BRANCH_NUMBER=86
CHROMEOS_RELEASE_BUILDER_PATH=eve-release/R63-10032.86.0
CHROMEOS_RELEASE_BUILD_NUMBER=10032
CHROMEOS_RELEASE_BUILD_TYPE=Official Build
CHROMEOS_RELEASE_CHROME_MILESTONE=63
CHROMEOS_RELEASE_DESCRIPTION=10032.86.0 (Official Build) stable-channel eve 
CHROMEOS_RELEASE_NAME=Chrome OS
CHROMEOS_RELEASE_PATCH_NUMBER=0
CHROMEOS_RELEASE_TRACK=stable-channel
CHROMEOS_RELEASE_VERSION=10032.86.0
DEVICETYPE=CHROMEBOOK
GOOGLE_RELEASE=10032.86.0
`

Model: `Pixelbook i5`

Architecture: `x86_64`

**Comment by uberhacker** [comment]
@dustinromey: This is the classic case of the wrong library location on x86_64 systems.  As a workaround, you can fix this with:
```
export LD_LIBRARY_PATH=/usr/local/lib64:/usr/local/lib
```
Until this package is updated to compile the library files in the correct location, this is what we need to do to fix it.  If you place this in your `~/.bashrc`, then you will have this set permanently in each login session.

**Comment by dustinromey** [comment]
Guess I'm still finding my way around with crew.

Almost worked perfectly. I did as you said, exited the shell and reentered, tried running the same commands and received the same warning about libjpeg.  

So I ran `crew install libjpeg`, allowed it to run, exited and reentered the shell and it ran fine.

Thanks for the help. Loving chromebrew.

**Comment by uberhacker** [comment]
Hey @dustinromey: Glad you like Chromebrew.  I think you just found a missing dependency in this package.  I'll make sure the next update includes `depends_on 'libjpeg'`.  Thanks for reporting.

**Comment by dustinromey** [comment]
I submitted a pull request for the change. Its a pretty simple one, but I wanted to go through it as there's a couple of packages I've thought of submitting (hugo, for example).

Thanks again, hope I was helpful.

**Comment by uberhacker** [comment]
Fixed in #1780.
